less/mixins.less

```diff
@@ -19,18 +19,33 @@
 @tablet: ~"(max-width: 768px)";
 @laptop: ~"(max-width: 1100px)";
 
+//super stretch goal - mobile first approach
+// @phone: ~"(min-width: 400px)";
+// @tablet: ~"(min-width: 768px)";
+// @laptop: ~"(min-width: 1100px)";
+
 //colors
 @primary-color: gray;
 @secondary-color: white;
+//task1 gray into black
+@black: black;
+//task7
+@japanese-laurel: green;
+//task8
+@fresh-eggplant: purple;
+//task9
+@web-orange: orange;
 
 // Mixins
+//task2
+@basefont: 62.5%;
```

Comment:
Great job setting up basefont and even greater job for using comments in Your code. Really awesome.